java/c/src/main/java/org/apache/arrow/c/Format.java

```diff
@@ -212,6 +212,10 @@ static String asString(ArrowType arrowType) {
         }
       case Utf8:
         return "u";
+      case Utf8View:
+        return "vu";
+      case BinaryView:
+        return "VZ";
```

Comment:
AFAIK the BinaryView format string is lowercase "vz"

Reply:
You're right.